src/lib/holocene/alert.svelte

```diff
@@ -23,7 +23,7 @@
 </script>
 
 <div
-  class="alert {intent} {className}"
+  class="flex alert {intent} {className}"
   class:bold
   role="alert"
```

Comment:
I wonder if we should just have this be determined by the component. I.e. `intent="error"` is `role="alert"` and all others are `role="status"` ?

Reply:
We will undoubtedly add a new info alert and forget to pass `role="status"` in the future

Reply:
That's fair. IMO, being in a box with these kinds of things wouldn't be the worst. Although another way we could do it is to just make the `role` prop required.

Reply:
What do you think about defaulting to `status` instead? That seems to be a pattern I see in other design systems.

Reply:
I think we should jump in the box and infer the role based on the intent. The second we need to break out of it we can update the component to accept a `role` prop to override the inferred ones. I think we'll have generally more accessible components with that calculated and it will automagically add accessibility in areas we already use those components.

I would say it should be like...
```
typescript
'warning' | 'caution' | 'error'  - alert
'success' - status
'info' - no role
```

Reply:
Hmm I guess we could go in the box, but still allow it to be overridden? 

There are examples in Cloud (e.g.audit logging) where the error is loaded with the rest of the page content and isn't something that is being updated based on actions or changing data, etc. In those cases we want the `error` `Alert` variant, but it shouldn't be `role="alert"`.

Reply:
Ahh ok, that's good to know. Sounds good to me 👍

Reply:
Updated [here](https://github.com/temporalio/ui/pull/1595/commits/706f9fd64c253d61653f1a3893179a60e9e866bd) ✅ with the following based on our use cases

```
'error' - alert
'success' | 'warning' | 'caution' - status
'info' - no role
```